tools/cockpit.spec

```diff
@@ -109,7 +109,9 @@ Recommends: (cockpit-docker if /usr/bin/docker)
 Recommends: (cockpit-networkmanager if NetworkManager)
 Recommends: (cockpit-storaged if udisks2)
 Recommends: cockpit-packagekit
+%if 0%{?rhel} && 0%{?centos} == 0
```

Comment:
this condition is pointless as it can never be rhel nor centos, see https://github.com/cockpit-project/cockpit/blob/master/tools/cockpit.spec#L104